gubernator/filters_test.py

```diff
@@ -45,6 +45,17 @@ def test_linkify(self):
     def test_slugify(self):
         self.assertEqual('k8s-test-foo', filters.do_slugify('[k8s] Test Foo'))
 
+    def test_testcmd_unit(self):
+        pkg = 'k8s.io/kubernetes/pkg/api/errors'
+        name = 'TestErrorNew'
+        expected = 'go test -v %s -run %s$' % (pkg, name)
```

Comment:
Fixed.

On Wed, Jun 8, 2016 at 6:55 PM, Joe Finney <notifications@github.com> wrote:

> In gubernator/filters_test.py
> <https://github.com/kubernetes/test-infra/pull/128#discussion_r66372287>:
>
> > @@ -45,6 +45,17 @@ def test_linkify(self):
> >      def test_slugify(self):
> >          self.assertEqual('k8s-test-foo', filters.do_slugify('[k8s] Test Foo'))
> >
> > +    def test_testcmd_unit(self):
> > +        pkg = 'k8s.io/kubernetes/pkg/api/errors'
> > +        name = 'TestErrorNew'
> > +        expected = 'go test -v %s -run %s$' % (pkg, name)
>
> generic testing blech: I'd rather you just copy the strings a few times
> than duplicate logic between the test and the code under test.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/test-infra/pull/128/files/eea1154ddbd157d20e4ac81a1cfde3d3ea10b0d5#r66372287>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAM7nD7nXkpXQnM-l7IJ4UfNba6BFMnuks5qJ3KjgaJpZM4Ixjp2>
> .
>